src/oauth2-client/service.ts

```diff
@@ -100,9 +100,39 @@ export async function getOAuth2ClientFromBody(ctx: Context): Promise<OAuth2Clien
 
 }
 
+export async function create(client: Omit<OAuth2Client, 'id'>, clientSecret: string, redirectUris: string[]): Promise<OAuth2Client> {
+
+  const query = 'INSERT INTO oauth2_clients SET ?';
+  const params = {
+    client_id: client.clientId,
+    client_secret: await bcrypt.hash(clientSecret, 12),
```

Comment:
Maybe I'm missing something, but does `clientSecret` need to be a param to this function? Should this value be available at `client.clientSecret`?

Reply:
This is definitely wrong